cartographer/mapping_3d/sparse_pose_graph.cc

```diff
@@ -536,7 +535,7 @@ transform::Rigid3d SparsePoseGraph::GetLocalToGlobalTransform(
 
 std::vector<std::vector<int>> SparsePoseGraph::GetConnectedTrajectories() {
   common::MutexLocker locker(&mutex_);
```

Comment:
Remove since `trajectory_connectivity_` is thread-safe? Same in 2D.

Reply:
Done